protostar/protostar_toml/protostar_toml_version_checker.py

```diff
@@ -0,0 +1,48 @@
+from typing import Any
+
+from protostar.protostar_exception import ProtostarException
+from protostar.protostar_toml.io.protostar_toml_reader import ProtostarTOMLReader
+from protostar.utils import VersionManager
+
+
+class ProtostarTOMLVersionChecker:
+    SKIP_FOR = ["init", "upgrade"]
+
+    def __init__(
+        self,
+        protostar_toml_reader: ProtostarTOMLReader,
+        version_manager: VersionManager,
+    ):
+        self._protostar_toml_reader = protostar_toml_reader
+        self._version_manager = version_manager
+
+    def run(self, args: Any):
```

Comment:
```suggestion
    def run(self, command: str):
```

Reply:
Business logic should be independent of the IO code.